src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/ScreenRecorder.cs

```diff
@@ -291,11 +262,16 @@ private IEnumerator StartRecordingCoroutine(string referenceSessionId)
 
                 Directory.CreateDirectory(stateRecordingsDirectory);
 
+                var prefix = referenceSessionId != null ? "replay" : "recording";
+                var pf = _currentSessionId.Substring(Math.Max(0, _currentSessionId.Length - 6));
+                var postfix = referenceSessionId != null ? pf + "_" + referenceSessionId.Substring(Math.Max(0, referenceSessionId.Length - 6)) : pf;
+                var dateTimeString =  System.DateTime.Now.ToString("MM-dd-yyyy_HH.mm");
```

Comment:
Nice! I suppose a next step for us (i'll track this somewhere) is to introduce some name indexing on the backend.